projects/engine/src/engine/plugin-engine.ts

```diff
@@ -82,9 +83,9 @@ export class PluginEngine<T extends ApiMap> extends AbstractPluginEngine {
     this.register(Object.keys(plugins).map(key => plugins[key]))
   }
 
-  public get actives(): string[] {
-    return Object.keys(this.methods)
-  }
+  // public get actives(): string[] {
```

Comment:
shouldn't that stay?

Reply:
Ah I forgot to remove the comments. No it's defined above with `protected actives: string[] = []`